test/express-compression.test.js

```diff
@@ -2,7 +2,7 @@
 
 const express = require('express');
 const request = require('supertest');
-const compressionMiddleware = require('../express-compression');
+const { compressionMiddleware } = require('../index');
```

Comment:
I would keep this import pointing to `../express-compression` since this is the specific test for that file.

The way this module exports its functions is a separate question.